ci-operator/config/openshift/runtime-utils/openshift-runtime-utils-master.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: "7"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+promotion:
+  name: "4.3"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 12Gi
+    requests:
+      cpu: "3"
+      memory: 8Gi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+test_binary_build_commands: make test
+tests:
+- as: verify
+  commands: |
+    make verify
+  container:
+    from: bin
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-unit JUNITFILE=/tmp/artifacts/unit_report.xml
+  container:
+    from: test-bin
```

Comment:
why isn't this also just "from: bin"?  or even "from src"?

Reply:
moved to `from: bin`